05-running-typedb-cluster/02-kubernetes.md

```diff
@@ -170,6 +170,7 @@ Configurable settings for Helm package include:
 
 | Key                 | Default value    | Description
 | :-----------------: | :---------------:| :------------------------------------------------------------------------------------------------: |
+| `name`              | `null`           | Used for naming deployed objects replacing the Helm release name used if this is not provided      |
```

Comment:
It's much better!

However this is even clearer still I think:

> Used for naming deployed objects. When not provided, the Helm release name will be used instead

What do you think?